AppCenter/AppCenter/Internals/Util/MSDispatcherUtil.h

```diff
@@ -3,6 +3,12 @@
 
 #import <Foundation/Foundation.h>
 
+#define MS_DISPATCH_SELECTOR(declaration, object, selectorName, ...)                                                                       \
+  ({                                                                                                                                       \
+    (declaration [object methodForSelector:NSSelectorFromString(selectorName)])(object, NSSelectorFromString(selectorName),                \
```

Comment:
Please make it multiline like in original macro

Reply:
Yes, sorry @ilyashumihin my example was really minimal.